lib/Db/TimelineQuery.php

```diff
@@ -11,7 +11,8 @@ class TimelineQuery
 {
     use TimelineQueryAlbums;
     use TimelineQueryDays;
-    use TimelineQueryFaces;
+    use TimelineQueryFaceRecognition;
+    use TimelineQueryRecognizePeople;
```

Comment:
Maybe `TimelineQueryPeopleFaceRecognition` and `TimelineQueryPeopleRecognize` for semantic grouping. I really shouldn't have done all of this trait stuff...